Signal/src/ViewControllers/AddToBlockListViewController.h

```diff
@@ -2,8 +2,8 @@
 //  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
 //
 
-#import <UIKit/UIKit.h>
+#import "SelectRecipientViewController.h"
 
-@interface AddToBlockListViewController : UIViewController
+@interface AddToBlockListViewController : SelectRecipientViewController
```

Comment:
I've pulled out a common base case `SelectRecipientViewController` from this view.  It can be used in any context where we're selecting a signal account, i.e. adding a user to a block list, adding a member to a new or existing group, etc.